cmd/dep/init.go

```diff
@@ -43,7 +43,7 @@ func (cmd *initCommand) Register(fs *flag.FlagSet) {}
 type initCommand struct{}
 
 func trimPathPrefix(p1, p2 string) string {
-	if filepath.HasPrefix(p1, p2) {
+	if strings.HasPrefix(p1, p2) {
 		return p1[len(p2):]
 	}
 	return p1
```

Comment:
[`filepath.HasPrefix` just calls `strings.HasPrefix`](https://golang.org/src/path/filepath/path_unix.go?s=710:747#L16). The reasons `filepath.HasPrefix` was deprecated are described [here](https://codereview.appspot.com/5712045). We should double-check this code this code for correctness. For now, I think using `strings.HasPrefix` is an improvement. It makes clear this is a pure string comparison. There isn't any added safety from using `filepath` (like you would get from using `filepath.Join` instead of `strings.Join`).